carbonj.service/src/main/java/com/demandware/carbonj/service/engine/ConfigServerUtil.java

```diff
@@ -0,0 +1,259 @@
+package com.demandware.carbonj.service.engine;
```

Comment:
Please add a license header to this file, otherwise the build will fail: 
https://github.com/salesforce/carbonj/runs/1731094229?check_suite_focus=true
`Caused by: org.gradle.api.GradleException: License violations were found: /home/runner/work/carbonj/carbonj/carbonj.service/src/main/java/com/demandware/carbonj/service/engine/ConfigServerUtil.java}`